modules/wildfly/src/main/modules/ogm/couchdb/module.xml

```diff
@@ -10,11 +10,12 @@
         <resource-root path="hibernate-ogm-couchdb-${project.version}.jar" />
     </resources>
     <dependencies>
-        <module name="org.hibernate" export="true" services="import" slot="${hibernate.ogm.module.slot}" />
+        <module name="org.hibernate" slot="${hibernate.ogm.module.slot}" />
         <module name="org.hibernate.hql" slot="${hibernate.hql.module.slot}" />
 
         <module name="javax.ws.rs.api" />
         <module name="javax.persistence.api" />
+        <module name="javax.transaction.api" />
```

Comment:
P.S. a legit usage of export is to have the OGM main module to export the ORM (vanilla) one.
We're pretty sure nobody would want to use OGM without ORM on his classpath :)
Likewise, for Hibernate Search to export Apache Lucene since its types are exposed on its public API.